scripts/index.js

```diff
@@ -345,11 +345,26 @@ metaHeader.addEventListener('click', toggleMenuMeta);
 var versionSelect = query('.site-menu .meta select');
 if (versionSelect) {
   versionSelect.addEventListener('change', function () {
-    var dest = attr(versionSelect, 'data-basepath') + '/' + versionSelect.value;
+
+    var dest = attr(versionSelect, 'data-basepath') + '/' + versionSelect.value;  
+
+    var basePath = attr(versionSelect, 'data-basepath');
+    var pathRegex = new RegExp('^' + escapeRegExp(basePath) + '\\/[\\w\\.]+\\/(?<relativePath>.*)');
+
+    var match = pathRegex.exec(location.pathname);
+    if (match !== null) {
+      dest = dest + '/' + match.groups.relativePath;
+    }
```

Comment:
While the Regex works, I think it is a bit overkill to get the site path.

I would add the baseUrl to the select element, which gives us something like `/manual/7.14/`, in https://github.com/camunda/camunda-docs-theme/blob/master/layouts/partials/menu.html#L47 : `base-url="{{ $.Site.BaseURL }}"`

With this, splitting becomes much easier 
```suggestion
    var basePath = attr(versionSelect, 'base-url');
    var result = location.href.split(basePath)[1];
    if (result) {
      dest = dest + '/' + result;
    }
```

Reply:
Thanks, I'll give it a try. I agree it's much better without a regex, but couldn't quite make the connection that this code can easily access the version of the docs it is executed in (or the full base url for that matter).